src/style.css

```diff
@@ -1 +1,72 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-bar__page {
+  font-weight: 300;
+  font-style: normal;
+  box-sizing: border-box;
+}
+
+.search-bar__form {
+  text-decoration: none;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.search-bar__form--big {
+  margin-top: 20px;
+}
+
+.search-bar__form--small {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  color: #3d4e61;
+  outline: none;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 5px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. In this case, you've specified Arial and Helvetica as fallbacks, which is good practice, but it's always a good idea to end with a generic family name such as 'sans-serif' to ensure maximum compatibility.